src/coprocessor/dag/executor/scanner.rs

```diff
@@ -81,9 +98,12 @@ impl<S: Snapshot> Scanner<S> {
     pub fn reset_range(&mut self, range: KeyRange, store: &SnapshotStore<S>) -> Result<()> {
         self.range = range;
         self.no_more = false;
+        unsafe {
+            self.seek_key.set_len(0);
```

Comment:
I think we should add comments. seek_key is a vector of primitive type u8, so that it is safe to use `set_len()` instead of `clear()` to skip the drop for every element. Also have you benchmarked the difference between the clear() version and set_len() version? I guess compiler is smart enough to eliminate the useless loop.

Reply:
Here I adopted @zhangjinpeng1987 's suggestion to use set_len here. Personally I don't suggest writing unsafe code anywhere, but hopefully it might improve the performance. I'll test it's improvement later.